packages/website/src/components/lib/jsonSchema.ts

```diff
@@ -8,6 +8,11 @@ const defaultRuleSchema: JSONSchema4 = {
   enum: ['off', 'warn', 'error', 0, 1, 2],
 };
 
+// https://github.com/microsoft/TypeScript/issues/17002
+function isArray(arg: unknown): arg is readonly unknown[] {
+  return isArray(arg);
+}
```

Comment:
```suggestion
function isArray(arg: unknown): arg is readonly unknown[] {
  return Array.isArray(arg);
}
```

as is i think this is an infinite loop

Reply:
Hahahaha "select all occurrences in the file and replace" did not even notice.